### DISCUSSION TITLE
ı get No module named 'giskard.rag'; 'giskard' is not a package eror.

### DISCUSSION THREAD

**Discussion by klncgty** [Giskard-AI/giskard-oss#1981] [Q&A] [answered]
when ı using this framework from giskard.rag import generate_testset, KnowledgeBase
ı get ModuleNotFoundError: No module named 'giskard.rag'; 'giskard' is not a package this error.

**Comment by alexcombessie** [comment, marked as answer]
Hey @klncgty - Just to check, are you sure that you have run `pip install giskard[llm]` in your Python virtual environment? 

Note the `[llm]` option which is required for `giskard.rag` to be available

**Reply by klncgty** [reply]
Thank You quick replay but i installed  this package.  again i faced this error.

**Reply by alexcombessie** [reply]
Could you start again from a clean Python virtual environment and send us the console output of the command `pip install giskard[llm]` ?

**Reply by klncgty** [reply]
thanks it works.

**Comment by VyvaHart** [comment]
Hi @klncgty 

Just try `pip install giskard[llm]`

Good Luck!

**Reply by eryk-mazus** [reply]
got the same error today, installed with `giskard[llm]`
